pmultiqc/modules/quantms/quantms.py

```
@@ -1838,6 +2015,261 @@ def getSpectrumCountAcrossRep(condition_count_dict: dict):
+            log.info("{}: Aggregating MzIdentML file {}...".format(datetime.now().strftime("%H:%M:%S"), m))
+
+            mzid_dict = list()
+            for mzid_tmp in mzid_data:
+                mzid_tmp_part = {k: v for k, v in mzid_tmp.items() if k not in ["SpectrumIdentificationItem"]}
+                for spectrum_item in mzid_tmp.get("SpectrumIdentificationItem", []):
+                    spectrum_item_part = {k: v for k, v in spectrum_item.items() if k not in ["PeptideEvidenceRef", "PeptideSequence"]}
+                    if spectrum_item_part.get("rank", None) == 1 and \
+                        spectrum_item_part.get("peptide passes threshold", "true") == "true" and \
+                            spectrum_item.get("passThreshold", False):
+                        for peptide_ref in spectrum_item.get("PeptideEvidenceRef", []):
+                            if "name" in peptide_ref.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)